kuma/wiki/models.py

```diff
@@ -380,6 +380,47 @@ def get_summary_html(self, *args, **kwargs):
     def get_summary_text(self, *args, **kwargs):
         return self.get_summary(strip_markup=True)
 
+    @classmethod
+    def from_url(cls, url, required_locale=None, id_only=False,
+                 check_host=True):
```

Comment:
I can see how ``required_locale`` and ``check_host`` might be useful in the future, but not in the context of this bug.  I'd lean toward removing them and the alternate code paths, and maybe linking to the same function in the kitsune repo, so they can be added back if the need arises.